.env.example

```diff
@@ -1,9 +1,4 @@
-REACT_APP_AUTHENTICATION_CLIENT_ID='editor-dev'
 REACT_APP_AUTHENTICATION_URL='http://localhost:9001'
-REACT_APP_API_ENDPOINT='http://localhost:3009'
-REACT_APP_GOOGLE_TAG_MANAGER_ID=''
-REACT_APP_PLAUSIBLE_DATA_DOMAIN=''
-REACT_APP_PLAUSIBLE_SOURCE=''
 REACT_APP_SENTRY_DSN=''
 REACT_APP_SENTRY_ENV='local'
 PUBLIC_URL='http://localhost:3011'
```

Comment:
Some of these env vars are still referenced in various places throughout the codebase, so we probably need to either remove those references, or not delete them quite yet 🤔 
e.g. REACT_APP_AUTHENTICATION_CLIENT_ID is referenced in `deploy.yml` and `userManager.js` etc. and the `userManager` is referenced in `login.js`, `LogoutButton` etc., but I'm not sure whether these are still being used 🤔 
I think the plausible vars are just being used in the github workflows so those references can probably go without too much trouble